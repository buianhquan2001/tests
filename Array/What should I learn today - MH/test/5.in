5
23
22 21 15 1 11 16 21 7 13 8 14 10 21 1 14 8 12 20 7 25 5 1 14 
5
9 17 16 9 24 
3
18 3 5 
22
25 8 19 21 14 14 3 20 16 25 5 1 15 24 14 6 22 6 20 17 2 23 
21
17 3 23 12 25 14 19 8 4 24 19 4 25 25 25 13 9 15 8 1 20 
