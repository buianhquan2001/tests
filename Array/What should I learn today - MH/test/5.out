english
english
math
math
math
